pytest_html/plugin.py

```diff
@@ -490,9 +491,12 @@ def generate_summary_item(self):
             __name__, os.path.join("resources", "main.js")
         ).decode("utf-8")
 
+        self.title = os.path.basename(self.logfile)
```

Comment:
With the default above (the suggestion), you can remove this line.

Reply:
Done